tests/integrations/test_otel_drop_in.py

```diff
@@ -0,0 +1,38 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2024 Datadog, Inc.
+
+from utils import weblog, interfaces, context, missing_feature, scenarios, features
+
+
+@features.f_otel_interoperability
+@missing_feature(condition=context.library != "java", reason="Endpoint is not implemented on weblog")
```

Comment:
Could we define this in the manifest files?

Reply:
Yes, good point: a7415c75f